workloads/sdn2ovn/run.sh

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env bash
+set -x
+
+source env.sh
+
+# _wait_for <resource> <resource_name> <desired_state> <timeout in minutes>
+_wait_for(){
+    echo "Waiting for $2 $1 to be $3 in $4 Minutes"
+    oc wait --for=condition=$3 --timeout=$4m $1 $2
+}
+_reboot(){
```

Comment:
As per recommendations you should reboot using AWS console, not by drain and cordoning.

Reply:
Also cordon should be before drain anyway, no?

Reply:
I'm checking if the cordon/drain procedure is even needed

Reply:
Is it okay to reboot nodes using debug command ? 

`oc debug  node/<NODE> -- chroot /host shutdown -r -t 3`

Reply:
Yes that should be good.